### PR TITLE
Update deployment-distributed.adoc

### PR DESCRIPTION
**Pull request by anilkumar29**

### CODE REVIEW

File: docs/modules/ROOT/pages/deployment-distributed.adoc [hunk 1]
```diff
@@ -1,8 +1,6 @@
-= Distributed deployment
+= Distributed multi-cluster deployment
 
-Currently not supported but on the roadmap.
-
-For those users who want to scale up the deployment and/or follow a more cloud-native approach using microservices that are easier to manage.
+The initial release of CMOS only supports single-cluster deployment; however, support for distributed multi-cluster deployment is the roadmap.
```
Comment (maria-robobug) [inline docs/modules/ROOT/pages/deployment-distributed.adoc:3]:
```suggestion
The initial release of CMOS only supports single-cluster deployment; however, support for distributed multi-cluster deployment is on the roadmap.
```

**Review by maria-robobug — APPROVED** [review]
👍 Just one minor comment.

File: docs/modules/ROOT/pages/deployment-distributed.adoc [hunk 2]
```diff
@@ -1,8 +1,6 @@
-= Distributed deployment
+= Distributed multi-cluster deployment
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/deployment-distributed.adoc:1]: "multi-cluster" suggests that 0.1 only supports one CBS cluster, when we support multiple (all monitored from the same CMOS instance) - we should make it clearer what "cluster" refers to here

Comment (anilkumar29) [reply docs/modules/ROOT/pages/deployment-distributed.adoc:1]: @markspolakovs make sense. I'll update it. Thanks!

**Pull request merged**